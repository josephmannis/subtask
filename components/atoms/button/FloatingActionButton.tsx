import React from 'react';
import { CircleButton } from './styles';

interface FloatingActionButtonProps {
    onPress: () => void;
}

const FloatingActionButton: React.FC<FloatingActionButtonProps> = props => {
    return (
        <CircleButton onPress={props.onPress}>
            
        </CircleButton>
    )
}

export default FloatingActionButton;