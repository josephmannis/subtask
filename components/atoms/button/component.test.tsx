import React from 'react';
import renderer from 'react-test-renderer';
import FloatingActionButton from './FloatingActionButton';

test('Renders', () => {
    const tree = renderer.create(<FloatingActionButton onPress={() => console.log('hi')}/>).toJSON();
    expect(tree).toMatchSnapshot()
})
