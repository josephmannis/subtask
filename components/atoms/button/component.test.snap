// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders 1`] = `
<View
  style={
    Array [
      Object {
        "bottom": 30,
        "position": "absolute",
        "right": 30,
      },
    ]
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#20A3FF",
        "borderRadius": 150,
        "elevation": 12,
        "height": 68,
        "justifyContent": "center",
        "opacity": 1,
        "shadowColor": "#20A3FF",
        "shadowOpacity": 0.3,
        "shadowRadius": 12,
        "width": 68,
      }
    }
  >
    <Image
      size="small"
      source={1}
      style={
        Array [
          Object {
            "aspectRatio": 1,
            "height": 20,
            "resizeMode": "contain",
            "width": 20,
          },
        ]
      }
    />
  </View>
</View>
`;
